lib/elixir/pages/typespecs.md

```diff
@@ -124,7 +124,9 @@ Built-in type           | Defined as
 `arity()`               | `0..255`
 `as_boolean(t)`         | `t`
 `binary()`              | `<<_::_*8>>`
+`nonempty_binary()`     | `<<_::8, _::_*8>>`
 `bitstring()`           | `<<_::_*1>>`
+`nonempty_bitstring()`  | `<<_::1, _::_*1>>`
```

Comment:
Let's keep it alphabetically ordered!

Reply:
They are not alphabetically ordered, `mfa/0` is after `nonempty*`. Should I fix that, or keep these close to their non-`nonempty` variant?

Reply:
I see, let’s keep it as is then. :)